docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -405,18 +463,35 @@ Check out our [advanced alerting tutorial](http://grafana.com/tutorials/alerting
 
 {{< /docs/ignore >}}
 
+<!-- INTERACTIVE page step10.md END -->
+
+<!-- INTERACTIVE page finish.md START -->
+
 ## Summary
 
+<!-- INTERACTIVE ignore START -->
+
 In this tutorial you learned about fundamental features of Grafana. To do so, we ran several Docker containers on your local machine. When you are ready to clean up this local tutorial environment, run the following command:
 
 ```
 docker-compose down -v
 ```
+<!-- INTERACTIVE ignore END -->
+
+{{< docs/ignore >}}
+
+In this tutorial you learned about fundamental features of Grafana. 
+
+{{< /docs/ignore >}}
+
+<!-- INTERACTIVE ignore START -->
 
 ### KillerCoda sandbox environment (completed tutorial)
 
 Do you want to see the finished result? Check out our [completed KillerCoda sandbox environment](https://killercoda.com/grafana-labs/course/full-stack/tutorial-enviroment-completed) containing the entire demo with dashboards, checks, and data sources configured.
```

Comment:
I would just delete this section as it makes sense for us to EOL those old sandboxes to make way for these